README.md

```diff
@@ -1402,12 +1403,12 @@ $ travis status -qpx && cap deploy
 
 ### Pro and Enterprise
```

Comment:
Let's replace `Pro and Enterprise` in this title with `Travis CI and Travis CI Enterprise` - what do you think?